@kiva/kv-components/vue/KvCommentsAdd.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<div class="tw-flex tw-flex-col">
+	<div class="kv-comments-add tw-flex tw-flex-col">
```

Comment:
Is this class needed?

Reply:
No. It's slightly more performant to use class selectors in combination with attribute selectors (which is what `:deep()` turns into) so I was thinking of trying that here, but ended up not bothering and forgot to remove the class.